### ISSUE TITLE
tests fail on armhf

### ISSUE THREAD

**Issue by aviau** [go-asn1-ber/asn1-ber#5]
Hello,

The Debian CI found that tests are failing on armhf, it looks like a simple bug:

```
src/gopkg.in/asn1-ber.v1/length_test.go:83: constant 9223372036854775807 overflows int
src/gopkg.in/asn1-ber.v1/length_test.go:137: constant 9223372036854775807 overflows int
```

Detailed logs: https://reproducible.debian.net/rb-pkg/unstable/armhf/golang-gopkg-asn1-ber.v1.html

**Comment by johnweldon** [comment]
Thank you @aviau

**Comment by rogers0** [comment]
Any update on this?
Now it becomes RC issue [0] for Debian, which means if not fixing it, this package will be removed from stretch release [1].
So please kindly help. Thank you!

[0] https://bugs.debian.org/860633
[1] https://udd.debian.org/cgi-bin/autoremovals.cgi

**Comment by liggitt** [comment]
published in [v1.2](https://github.com/go-asn1-ber/asn1-ber/releases/tag/v1.2)
